### PR TITLE
tests: Update example using new utils

### PR DESCRIPTION
**Pull request by robertbaldyga**
Needs: https://github.com/Open-CAS/test-framework/pull/62

### CODE REVIEW

File: test/functional/tests/example/example_test.py [hunk 1]
```diff
@@ -19,37 +19,67 @@ def setup_module():
...
+        file1 = File.create_file("example_file")
+        file1.write("Test file\ncontent line\ncontent")
+    with TestRun.step("Read file content"):
+        content_before_change = file1.read()
+        TestRun.LOGGER.info(f"File content: {content_before_change}")
+    with TestRun.step("Replace single line in file"):
+        fs_utils.replace_in_lines(file1, 'content line', 'replaced line')
+    with TestRun.step("Read file content and check if it changed"):
+        content_after_change = file1.read()
+        assert content_before_change != content_after_change
```
Comment (katlapinka) [inline test/functional/tests/example/example_test.py]: please do not use assert but TestRun.fail()

File: test/functional/tests/example/example_test.py [hunk 2]
```diff
@@ -19,37 +19,67 @@ def setup_module():
...
+
+    with TestRun.step("Make copy of the file and check if md5 sum matches"):
+        file2 = file1.copy('/tmp', force=True)
+        assert file1.md5sum() == file2.md5sum()
```
Comment (katlapinka) [inline test/functional/tests/example/example_test.py]: as before - use TestRun.fail()

File: test/functional/tests/example/example_test.py (hunk 1 above)
Comment (robertbaldyga) [reply test/functional/tests/example/example_test.py]: Done.

File: test/functional/tests/example/example_test.py (hunk 2 above)
Comment (robertbaldyga) [reply test/functional/tests/example/example_test.py]: Done.

**Pull request merged**